velox/functions/lib/SimpleComparisonMatcher.h

```diff
@@ -73,11 +77,17 @@ class ComparisonMatcher : public Matcher {
     VELOX_CHECK_EQ(2, inputMatchers_.size());
   }
 
+  // Checks if the given name specifies a comparison expression. Can be
+  // overriden to use different function names for Spark.
+  virtual bool exprNameMatch(const std::string& name) {
+    return name == prefix_ + "eq" || name == prefix_ + "lt" ||
+        name == prefix_ + "gt";
+  }
+
   bool match(const core::TypedExprPtr& expr) override {
```

Comment:
Perhaps move the implementation to cpp file if it is not a small function to be inlined. Ditto for the others.